apps/DensifyPointCloud/DensifyPointCloud.cpp

```diff
@@ -296,6 +298,23 @@ int main(int argc, LPCTSTR* argv)
 	// load and estimate a dense point-cloud
 	if (!scene.Load(MAKE_PATH_SAFE(OPT::strInputFileName)))
 		return EXIT_FAILURE;
+	if (!OPT::strMaskPath.empty()) {
+		if (OPTDENSE::nIgnoreMaskLabel < 0) {
```

Comment:
this check is probably not needed, as all you do here is setting the maskName variable, which will not be used if not requested; however can be saved and added to the MVS scene file; what do you think?